src/routes/dashboard.jsx

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2019-2024 The Tekton Authors
```

Comment:
New file only needs 2024, like the other new files?
```suggestion
Copyright 2024 The Tekton Authors
```

Reply:
The content is largely copied from the old route definitions in App.jsx which is why I kept the original date, just restructured to follow the new API

Reply:
although I realise now that I didn't do the same for the pipelines and triggers routes… 🙈 
I'll update them all for consistency. Thanks